app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
```

Comment:
Since both `NotVaccinatedError` and `OutdatedVaccineError` are subclasses of `VaccineError`, you can replace them by `VaccineError`:

```suggestion
        except VaccineError:
```

Don't forget to import `VaccineError` and remove the `NotVaccinatedError` and `OutdatedVaccineError`  imports.